routers/api/v1/repo/issue.go

```diff
@@ -272,12 +272,14 @@ func SearchIssues(ctx *context.APIContext) {
 			return
 		}
 
-		issuesOpt.ListOptions = db.ListOptions{
-			Page: -1,
-		}
-		if filteredCount, err = issues_model.CountIssues(ctx, issuesOpt); err != nil {
-			ctx.Error(http.StatusInternalServerError, "CountIssues", err)
-			return
+		if filteredCount == 0 {
```

Comment:
Under what circumstances is `filteredCount` `0` and why should we return the total number of issues then?
Can't we just delete this whole branch now?